[2024-05-13T19:22:34.674+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-05-13T19:22:34.737+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: Units_statistics_DAG.ETL_job scheduled__2024-05-13T19:00:00+00:00 map_index=3 [queued]>
[2024-05-13T19:22:34.763+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: Units_statistics_DAG.ETL_job scheduled__2024-05-13T19:00:00+00:00 map_index=3 [queued]>
[2024-05-13T19:22:34.764+0000] {taskinstance.py:2306} INFO - Starting attempt 2 of 4
[2024-05-13T19:22:34.795+0000] {taskinstance.py:2330} INFO - Executing <Mapped(_PythonDecoratedOperator): ETL_job> on 2024-05-13 19:00:00+00:00
[2024-05-13T19:22:34.808+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=1788) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-05-13T19:22:34.811+0000] {standard_task_runner.py:63} INFO - Started process 1791 to run task
[2024-05-13T19:22:34.821+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'Units_statistics_DAG', 'ETL_job', 'scheduled__2024-05-13T19:00:00+00:00', '--job-id', '56506', '--raw', '--subdir', 'DAGS_FOLDER/Units_dag.py', '--cfg-path', '/tmp/tmpbruq9emx', '--map-index', '3']
[2024-05-13T19:22:34.824+0000] {standard_task_runner.py:91} INFO - Job 56506: Subtask ETL_job
[2024-05-13T19:22:34.972+0000] {task_command.py:426} INFO - Running <TaskInstance: Units_statistics_DAG.ETL_job scheduled__2024-05-13T19:00:00+00:00 map_index=3 [running]> on host 117fd00757b6
[2024-05-13T19:22:35.217+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='abeer-araby' AIRFLOW_CTX_DAG_ID='Units_statistics_DAG' AIRFLOW_CTX_TASK_ID='ETL_job' AIRFLOW_CTX_EXECUTION_DATE='2024-05-13T19:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-05-13T19:00:00+00:00'
[2024-05-13T19:22:35.218+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-05-13T19:22:35.219+0000] {Units_dag.py:98} INFO - i: 30000, chunk: 10000
[2024-05-13T19:22:35.219+0000] {Units_dag.py:99} INFO - child token is::: eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJodHRwczovL2FwaS5hZmFxeS5zYS9hdXRoL2xvZ2luX2FzIiwiaWF0IjoxNzE1NjI4MDA1LCJleHAiOjE3MTgyMjAwMDUsIm5iZiI6MTcxNTYyODAwNSwianRpIjoiUTVsWWFFWTh1dmxkUW9VZyIsInN1YiI6IjVkNjNlNTFlMDNlZDUyOTJhNDJjMjRlNiIsInBydiI6IjI3MGVjZmM3ZWEzZWQ5MzdlYTg0OTM2MmEzYTUwOTEwYzZkOGNlNGYiLCJsb2dpbl9hc19pZCI6IjY1N2YwYWQ3M2Y5NzcyOTVjZjA1NTc2YSJ9.ZJG7FP7VsHLrMIahOzJATZnUnTORygvb_e6VKyJncgs
[2024-05-13T19:22:51.651+0000] {Units_dag.py:103} INFO - {'_id': '621751b500922748b2286cc5', 'icon': 'icons/icon.png', 'driver_id': '', 'tailer_id': '', 'profile': {'vehicle_type': '', 'vin': '', 'plate_number': '', 'model': '', 'year': 0, 'seats': 2, 'fuel_type': '91', 'start_operation': '', 'contract_milage': '', 'opening_milage': '', 'measurement_type': '', 'max_capacity': ''}, 'device_password': '***', 'device_type': 'gps', 'gateway': False, 'name': '866854051825647', 'operation_code': None, 'device_gateway': None, 'business_types': {'waste': False, 'transportation': True, 'water': False}, 'imei': '866854051825647', 'sim_number': '+966830020160204', 'sim_serial': '8996641520415664715', 'device_serial': '866854051825647', 'device': 'bce', 'user_id': '619b47e8a0de2d54c7741f56', 'created_by': '5e6655bf03ed52e1ab79d301', 'updated_at': '2024-04-18 11:18:59', 'created_at': '2022-02-24 12:36:53', 'custom_fields': [], 'creator': 'amr.ahmed', 'company': '', 'owner': 'general-commission-admin', 'counters': {'odometer_type': 'gps', 'odometer': None, 'engine_hours_type': 'acc', 'engine_hours': 0}, 'is_wasl_connected': 'Not Connected', 'iconUrl': 'https://api.afaqy.sa/images/units/icons/icon.png', 'hasCCTV': False}
[2024-05-13T19:22:52.329+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/elasticsearch/_sync/client/__init__.py:399: SecurityWarning: Connecting to 'https://dbelk1.afaqy.local:9200' using TLS with verify_certs=False is insecure
  _transport = transport_class(

[2024-05-13T19:22:52.379+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/elasticsearch/_sync/client/__init__.py:399: SecurityWarning: Connecting to 'https://dbelk2.afaqy.local:9200' using TLS with verify_certs=False is insecure
  _transport = transport_class(

[2024-05-13T19:22:52.812+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/elasticsearch/_sync/client/__init__.py:399: SecurityWarning: Connecting to 'https://dbelk3.afaqy.local:9200' using TLS with verify_certs=False is insecure
  _transport = transport_class(

[2024-05-13T19:22:52.843+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/urllib3/connectionpool.py:1103: InsecureRequestWarning: Unverified HTTPS request is being made to host 'dbelk1.afaqy.local'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#tls-warnings
  warnings.warn(

[2024-05-13T19:22:52.933+0000] {_transport.py:349} INFO - GET https://dbelk1.afaqy.local:9200/_cluster/health [status:200 duration:0.119s]
[2024-05-13T19:22:53.301+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/urllib3/connectionpool.py:1103: InsecureRequestWarning: Unverified HTTPS request is being made to host 'dbelk2.afaqy.local'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#tls-warnings
  warnings.warn(

[2024-05-13T19:22:54.731+0000] {_transport.py:349} INFO - PUT https://dbelk2.afaqy.local:9200/units_statistics_***/_bulk [status:200 duration:1.464s]
[2024-05-13T19:22:54.959+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-05-13T19:22:54.960+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-05-13T19:22:54.978+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=Units_statistics_DAG, task_id=ETL_job, run_id=scheduled__2024-05-13T19:00:00+00:00, map_index=3, execution_date=20240513T190000, start_date=20240513T192234, end_date=20240513T192254
[2024-05-13T19:22:55.038+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-05-13T19:22:55.078+0000] {taskinstance.py:3498} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-05-13T19:22:55.080+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
